crates/sui-indexer-alt/src/schema.rs

```diff
@@ -64,6 +72,30 @@ diesel::table! {
     }
 }
 
+diesel::table! {
```

Comment:
curious is this now in sync with migrations and thus can be generated by diesel cmd?